src/domain_factory/product/product_factory.py

```diff
@@ -62,7 +62,7 @@ def generate_product_id(self, id):
             A single product Id
         """
 
-        return 1000 + id
+        return id
```

Comment:
Why have you removed the 1000 in the front?  If you think that there shouldn't be numbers in front of IDs should you raise a fresh issue and make the change in product_factory.py and brokerage_account_factory.py

Reply:
added back. As discussed, any dependant will retrieve created productIds from the database